src/sage/structure/nonexact.py

```diff
@@ -19,7 +19,7 @@
     sage: R.<x> = PowerSeriesRing(QQ, default_prec=10)
     sage: R.default_prec()
     10
-    sage: cos(x)
+    sage: cos(x)                                                                        # optional - sage.symbolic
```

Comment:
Does using `x.cos()` bypass the symbolic cosine?

Reply:
Yes.

Reply:
Why not `needs sage.symbolic`?